templates/rapidsai-core/Devel.dockerfile.j2

```diff
@@ -36,7 +36,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     && rm -rf /var/lib/apt/lists/*
 {% endif %}
 
-RUN mkdir -p ${RAPIDS_DIR}/utils {{ "${GCC9_DIR}/lib64" if "centos" in os }}
+RUN mkdir -p ${RAPIDS_DIR}/utils {{ "${GCC9_DIR}/lib64" if ( "rocky" in os or "centos" in os ) }}
```

Comment:
```suggestion
RUN mkdir -p ${RAPIDS_DIR}/utils
```

Since the switch to `conda` compilers, this conditional statement is no longer needed and can therefore be removed.

Reply:
removed.